components/clarity-repl/src/repl/session.rs

```diff
@@ -1311,3 +1318,27 @@ mod tests {
         assert_eq!(session.handle_command("(at-block (unwrap-panic (get-block-info? id-header-hash u10000)) (contract-call? .contract get-x))")[0], green!("u1"));
     }
 }
+
+use reqwest;
+use std::future::Future;
+use tokio;
+
+pub fn nestable_block_on<F: Future>(future: F) -> F::Output {
+    let (handle, _rt) = match tokio::runtime::Handle::try_current() {
+        Ok(h) => (h, None),
+        Err(_) => {
+            let rt = tokio::runtime::Runtime::new().unwrap();
+            (rt.handle().clone(), Some(rt))
+        }
+    };
+    let response = handle.block_on(async { future.await });
+    response
+}
+
+async fn fetch_message() -> Result<String, reqwest::Error> {
+    const gist: &str =
+        "https://gist.githubusercontent.com/obycode/d10b5a40cd926275ff6bde07d2270f2a/raw/";
```

Comment:
@aravindgee could you help us and give us access to a storage bucket / cdn for these assets?

Reply:
Uploaded to one of our public buckets, here's the URL: https://storage.googleapis.com/hiro-public/assets/clarinet-egg.txt

[[FYI a better CDN for Hiro Assets is in the works](https://github.com/hirosystems/devops/issues/1048)]

How frequently are we looking to update this file?

Reply:
Branch updated with new URL - will merge!